transcom/mymove
# B 20724 int

## [B-20724](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983379)

## Summary

This task removes the Primes ability to update the Prime Estimated Weight and Previously Recorded Weight ( NTSRecordedWeight). These fields are now read only for the prime on update for NTS-Release shipments.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office app
2. Login as a TOO
3. Add an NTS-Release shipment to any move, and then approve it
4. Grab the Move and Shipment ID, as well as the etag required to update.
5. Log in to the Prime swagger API (all versions were updated)
6. Move the UpdateMTOShipment endpoint
7. Apply the proper JSON to update your shipment (you will need to create/acquire this. I used the debugger to copy an update json from prime sim and then used the new etag that was returned.)
8. Verify that you cannot update the Prime Estimated Weight or Previously Recorded Weight (NTSRecordedWeight)

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:
